src/main/java/com/doeiqts/pizzavoter/domain/Pizza.java

```diff
@@ -33,7 +34,7 @@ public Pizza(Size size, Crust crust, Sauce sauce) {
         toppingCount = 1.0;
     }
 
-    public Position addTopping(Topping topping, Position position) {
+    public Position addTopping(Topping topping, Position position, HttpServletRequest request) {
```

Comment:
You can't always get what you want, but yeah I don't like it either. Just was the best way to handle this quickly. That or we make a service class.